argocon-demos/2025-automated-resilience-using-argo-events-for-real-time-incident-remediation/README.md

```diff
@@ -0,0 +1,17 @@
+                                          # Automated Resilience: Using Argo Events for Real-Time Incident Remediation
+
+[![Pipekit Logo](https://raw.githubusercontent.com/pipekit/talk-demos/main/assets/images/pipekit-logo.png)](https://pipekit.io?utm_campaign=talk-demos)
+
+## The talk
+The talk recording [can be found here]().
```

Comment:
comment this line out for now. We'll add it back in once the videos are up next week